Sources/InstantiateStandard/Extensions.swift

```diff
@@ -26,6 +26,9 @@ extension IdentifierType {
 }
 
 #if os(iOS) || os(tvOS)
+
+    extension NSStoryboard.Name: IdentifierType {}
+    extension NSNib.Name: IdentifierType {}
```

Comment:
`NSStoryboard` and `NSNib` is macOS classes. 🤔

Reply:
Oh, my bad. Thanks for letting me know.